src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -112,7 +112,8 @@ object Cluster {
       instances = Set.empty,
       userJupyterExtensionConfig = clusterRequest.userJupyterExtensionConfig,
       autopauseThreshold = autopauseThreshold,
-      defaultClientId = clusterRequest.defaultClientId)
+      defaultClientId = clusterRequest.defaultClientId,
+      stopAfterCreation= clusterRequest.stopAfterCreation.getOrElse(false))
```

Comment:
i.e. map `stopAfterCreation` to false unless it was an optional `true`

PS. Will fix the typo after `stopAfterCreation`